google-home-community.groovy

```diff
@@ -59,6 +59,17 @@
 //   * May 20 2021 - Add a reverseDirection setting to the Open/Close trait to support devices that consider position
 //                   0 to be fully open
 //   * Jun 27 2021 - Log a warning on SYNC if a device is selected as multiple device types
+//   * Mar 05 2022 - Added supportsFanSpeedPercent trait for controlling fan by percentage
+//   * May 07 2022 - Add error handling so one bad device doesn't prevent reporting state of other devices
+//   * Jun 15 2022 - Fix a crash on trait configuration introduced by Hubitat 2.3.2.127
+//   * Jun 16 2022 - Fixed CameraStream trait to match the latest Google API.  Moved protocol support to the
+//                   driver level to accomodate different camera stream sources
+//                 - Added Arm/Disarm Trait
+//                 - Added ability for the app to use trait level pin codes retrieved from the device for
+//                   Arm/Disarm and Lock/Unlock
+//                 - Modified checkMfa to accept deviceInfo (instead of deviceType)
+//                   Added support for trait level pincodes, as well as returning the matching user position or
+//                   no match error value on pincode challenge in the order of trait -> device -> global -> null
```

Comment:
Nitpick: I would drop the bit about modifying `checkMfa` since it's not really a user-visible change.

Reply:
Agreed.